react/finger-exercises/redux/02 - react-redux/src/utils/utils.js

```diff
@@ -0,0 +1,5 @@
+export default function filterByName(arr, value) {
+  return arr.filter(function (a) {
+    return a.name.toLowerCase().indexOf(value.toLowerCase()) !== -1;
```

Comment:
you can write this as an arrow function
arr.filter(a => a.name ...)

Besides, I would write a more descriptive name of each element